features/FEATURE_BLE/ble/pal/MemorySecurityDB.h

```diff
@@ -207,6 +238,7 @@ class MemorySecurityDb : public SecurityDb {
         if (entry) {
             entry->state = ENTRY_WRITTEN;
             entry->peer_identity.irk = irk;
+            entry->flags.irk_stored;
```

Comment:
you're missing the;
= true
that's why it would be nice if we got rid of all the warnings because then any new warnings would be visible when you compile